packages/story-editor/package.json

```diff
@@ -0,0 +1,72 @@
+{
+	"name": "@web-stories-wp/story-editor",
+	"private": true,
+	"engines": {
+		"node": ">= 15",
+		"npm": "~7"
+	},
+	"exports": {
+		"default": "./src/index.js"
+	},
+	"main": "./src/index.js",
+	"type": "module",
+	"dependencies": {
```

Comment:
All of these can now be removed from the main `package.json` file in the plugin root (unless they're also used by dashboard of course).